jaxlib/BUILD

```diff
@@ -31,11 +31,24 @@ cc_library(
     ],
     features = ["-use_header_modules"],
     deps = [
-        "@com_google_absl//absl/base",
+        ":kernel_helpers_no_pybind11",
         "@pybind11",
     ],
 )
 
+cc_library(
+    name = "kernel_helpers_no_pybind11",
+    hdrs = ["kernel_helpers_no_pybind11.h"],
+    copts = [
+        "-fexceptions",
```

Comment:
If you don't use pybind11 in a module, you don't need these two copts or the `features` below. They exist mostly for enabling exceptions for Google builds which disable exceptions by default.

Reply:
I think we still need -fexceptions here because UnpackDescriptor throws. I removed the other copt and feature though.